<ui version="4.0" >
 <class>datetimeFrm</class>
 <widget class="QWidget" name="datetimeFrm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>424</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="atab" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Date &amp;&amp; Time</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" >
       <item row="0" column="0" >
        <spacer name="horizontalSpacer_7" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>52</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" >
        <layout class="QGridLayout" name="gridLayout" >
         <property name="verticalSpacing" >
          <number>16</number>
         </property>
         <item row="0" column="0" >
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1" >
          <widget class="QDateEdit" name="dateEdit" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2" >
          <spacer name="horizontalSpacer_2" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="3" >
          <widget class="QCalendarWidget" name="calendar" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>190</width>
             <height>210</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>190</width>
             <height>210</height>
            </size>
           </property>
           <property name="firstDayOfWeek" >
            <enum>Qt::Monday</enum>
           </property>
           <property name="gridVisible" >
            <bool>true</bool>
           </property>
           <property name="navigationBarVisible" >
            <bool>true</bool>
           </property>
           <property name="dateEditEnabled" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2" >
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3" >
        <layout class="QGridLayout" name="gridLayout_2" >
         <property name="verticalSpacing" >
          <number>16</number>
         </property>
         <item row="0" column="0" >
          <spacer name="horizontalSpacer_3" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1" >
          <widget class="QTimeEdit" name="timeEdit" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="minimumSize" >
            <size>
             <width>90</width>
             <height>0</height>
            </size>
           </property>
           <property name="displayFormat" >
            <string>HH:mm AP</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" >
          <spacer name="horizontalSpacer_4" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="3" >
          <widget class="WallClock" native="1" name="aclock" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>190</width>
             <height>190</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>190</width>
             <height>190</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="4" >
        <spacer name="horizontalSpacer_6" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>52</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2" >
        <spacer name="verticalSpacer_3" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="5" >
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <spacer name="horizontalSpacer_10" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>25</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="horizontalSpacer_5" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>128</width>
             <height>22</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="revertBtn" >
           <property name="font" >
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text" >
            <string>Revert</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveBtn" >
           <property name="font" >
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text" >
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>148</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="helpBtn" >
           <property name="minimumSize" >
            <size>
             <width>27</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>27</width>
             <height>27</height>
            </size>
           </property>
           <property name="font" >
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text" >
            <string>?</string>
           </property>
           <property name="icon" >
            <iconset resource="../syspref.qrc" >
             <normaloff>:/icons/tool/question.png</normaloff>:/icons/tool/question.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QToolButton" name="unlockBtn" >
       <property name="minimumSize" >
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="../syspref.qrc" >
         <normaloff>:/icons/tool/lock.png</normaloff>:/icons/tool/lock.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="checked" >
        <bool>false</bool>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lockLbl" >
       <property name="text" >
        <string>Click here to unlock and allow changes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WallClock</class>
   <extends>QWidget</extends>
   <header location="global" >clockwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../syspref.qrc" />
 </resources>
 <connections/>
</ui>
